hmri_proc_crtMask.m

```diff
@@ -0,0 +1,153 @@
+function [fn_maskTC,fn_meanTC] = hmri_proc_crtMask(fn_smwTC, opts)
+% Generating tissue specific mask for the SPM analysis.
+% The aim is to preselect, based on smoothed modulated posterior tissue 
+% probability maps, which voxels should be considered for further 
+% statistical analysis.
+% 
+% FORMAT
+%   [fn_maskTC,fn_meanTC] = hmri_proc_crtMask(fn_smwTC, opts)
+% 
+% INPUT
+% - fn_smwTC : filenames (cell array, 1 cell per TC, containing a char or 
+%             cell array) of the modulated warped tissue classes, 
+%             ex. the mwc1/2*.nii filenames in the 1st and 2nd cell.
+% - opts    : option structure
+%   .minTCp : minimal posterior probability value to be considerd [.2, def]
+%   .noOvl  : avoid overlap by assigning to tissue class with maximum
+%             posterior probability [1, def]
+%   .outPth : path to output folder, use that of the 1st image if it is 
+%             left empty ['', def]
+% 
+% OUTPUT
+% - fn_maskTC : filenames (char array) of the tissue specific masks
+% - fn_meanTC : filenames (char array) of the mean tissue class images
+% 
+% The core idea comes from Callaghan et al.
```

Comment:
Year, journal for Callaghan, et al.?

Reply:
2014 https://doi.org/10.1016/j.neurobiolaging.2014.02.008